### PR TITLE
Various fixes to btbridged

### LINKED ISSUE #2: btbridged stops  working when it starts opal
If I restart the process then it becomes functional again.

[114922032403,7] AST: PNOR LPC offset: 0x0e000000
[114922100756,5] PLAT: Using SuperIO UART
[114980667219,5] PLAT: Detected Palmetto platform
[114993811755,5] CENTAUR: Found centaur for chip 0x0 channel 1
[114996510097,5] CENTAUR:   FSI host: 0x0 cMFSI0 port 2
[114998666513,5] CENTAUR:   ChipID 0x80000001 [DD2.0]
[115001515171,5] PSI[0x000]: Found PSI bridge [working=1, active=0]
[917265479,5] BT: Interface intialized, IO 0x00e4
[2455220760,3] BT seq 0x00 cmd 0x24: Retry sending message
[3993730645,3] BT seq 0x00 cmd 0x24: Timeout sending message
[5532426390,3] BT seq 0x01 cmd 0x22: Retry sending message

### PR DESCRIPTION
**Pull request by cyrilbur-ibm**
Also threw in a simple Makefile improvement

### CODE REVIEW

**Comment by cyrilbur-ibm** [comment]
First commit fixes #2

File: btbridged.c [hunk 1]
```diff
@@ -273,7 +271,7 @@ static int method_send_message(sd_bus_message *msg, void *userdata, sd_bus_error
 		r = 0;
 		goto out;
 	}
-	r = sd_bus_message_read(msg, "yyyyy", &netfn, &lun, &seq, &cmd, &cc);
+	r = sd_bus_message_read(msg, "yyyyy", &seq, &netfn, &lun, &cmd, &cc);
```
Comment (causten) [inline btbridged.c]: Does this make sense to be a message_read?  Should it be a message_append?  This should be yyyyay.  The code will send data packet to the host.

**Comment by causten** [comment]
I have tested and all my issues are fixed.  Thanks!

**Comment by williamspatrick** [comment]
Gave minor comments about makefile in mailing list.  This has been tested by Chris and had sufficient time for review.

**Comment by williamspatrick** [comment]
I probably wouldn't normally merge this project, but need to get the recipe updated for barreleye image build.  Merging...

**Pull request merged**